### PR TITLE
Try to use 2x the number of cores.

### PR DESCRIPTION
**Pull request by jpakkane**
Just an experiment to see what would happen. Ninja is probably faster, msbuild might or might not be slower.

### CODE REVIEW

File: run_project_tests.py [hunk 1]
```diff
@@ -503,6 +503,9 @@ def _run_tests(all_tests, log_name_base, extra_args):
         print('Could not determine number of CPUs due to the following reason:' + str(e))
         print('Defaulting to using only one process')
         num_workers = 1
+    # Due to Ninja deficiency, almost 50% of build time
```
Comment (polarina) [inline run_project_tests.py]: Can you clarify what kind of deficiency is being referred to?

Comment (jpakkane) [reply run_project_tests.py]: Basically it's #2082, but I'll add a better description if we actually consider this for merging (as opposed to Ninja fixing the actual issue).

**Comment by jpakkane** [comment]
About 10-15 minutes faster on both Travis and Appveyor. Not fabulous, but not too shabby either.

**Comment by nirbheek** [comment]
Let's merge this I guess. We can check if we need to change this again when we fix up and merge #2082.

**Pull request merged**